src/engraving/libmscore/articulation.cpp

```diff
@@ -676,54 +677,35 @@ double Articulation::mag() const
-           || _symId == SymId::articMarcatoTenutoAbove || _symId == SymId::articMarcatoTenutoBelow
-           || _symId == SymId::articTenutoAccentAbove || _symId == SymId::articTenutoAccentBelow;
-}
-
-//---------------------------------------------------------
-//   isLuteFingering
-//---------------------------------------------------------
-
-bool Articulation::isLuteFingering() const
-{
-    return _symId == SymId::stringsThumbPosition
-           || _symId == SymId::luteFingeringRHThumb
-           || _symId == SymId::luteFingeringRHFirst
-           || _symId == SymId::luteFingeringRHSecond
-           || _symId == SymId::luteFingeringRHThird;
+    _isDouble = _symId == SymId::articMarcatoStaccatoAbove || _symId == SymId::articMarcatoStaccatoBelow
+                || _symId == SymId::articTenutoStaccatoAbove || _symId == SymId::articTenutoStaccatoBelow
+                || _symId == SymId::articAccentStaccatoAbove || _symId == SymId::articAccentStaccatoBelow
+                || _symId == SymId::articMarcatoTenutoAbove || _symId == SymId::articMarcatoTenutoBelow
+                || _symId == SymId::articTenutoAccentAbove || _symId == SymId::articTenutoAccentBelow;
+
+    _isTenuto = _symId == SymId::articTenutoAbove || _symId == SymId::articTenutoBelow
+                || _symId == SymId::articMarcatoTenutoAbove || _symId == SymId::articMarcatoTenutoBelow
+                || _symId == SymId::articTenutoAccentAbove || _symId == SymId::articTenutoAccentBelow;
+
+    _isStaccato = _symId == SymId::articStaccatoAbove || _symId == SymId::articStaccatoBelow
+                  || _symId == SymId::articMarcatoStaccatoAbove || _symId == SymId::articMarcatoStaccatoBelow
+                  || _symId == SymId::articTenutoStaccatoAbove || _symId == SymId::articTenutoStaccatoBelow
+                  || _symId == SymId::articAccentStaccatoAbove || _symId == SymId::articAccentStaccatoBelow;
+
+    _isAccent = _symId == SymId::articAccentAbove || _symId == SymId::articAccentBelow
+                || _symId == SymId::articAccentStaccatoAbove || _symId == SymId::articAccentStaccatoBelow;
+
+    _isMarcato = _symId == SymId::articMarcatoAbove || _symId == SymId::articMarcatoBelow
+                 || _symId == SymId::articMarcatoStaccatoAbove || _symId == SymId::articMarcatoStaccatoBelow
+                 || _symId == SymId::articMarcatoTenutoAbove || _symId == SymId::articMarcatoTenutoBelow;
+
+    _isLuteFingering = _symId == SymId::articMarcatoStaccatoAbove || _symId == SymId::articMarcatoStaccatoBelow
+                       || _symId == SymId::articTenutoStaccatoAbove || _symId == SymId::articTenutoStaccatoBelow
+                       || _symId == SymId::articAccentStaccatoAbove || _symId == SymId::articAccentStaccatoBelow
+                       || _symId == SymId::articMarcatoTenutoAbove || _symId == SymId::articMarcatoTenutoBelow
+                       || _symId == SymId::articTenutoAccentAbove || _symId == SymId::articTenutoAccentBelow;
```

Comment:
Looking at the old definition of `isLuteFingering()`, this doesn't seem right to me...

Reply:
Shit, wrong copy-paste. Glad you noticed